aten/src/ATen/native/quantized/QTensor.cpp

```diff
@@ -151,5 +151,27 @@ Tensor quantized_clone(const Tensor& self) {
   return dst;
 }
 
+bool quantized_equal(const Tensor& self, const Tensor& other) {
```

Comment:
We need a check for the scheme as well:

```python
if (self.q_scheme() != other.q_scheme) return false;
```

Reply:
I think I'm actually going to define comparisons on Quantizer and delegate to those. That way we can have specific logic for each scheme to check equivalence (i.e. comparing scale vs scales for per_channel)